presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1318,8 +1318,8 @@ public void testDataMappingSmokeTest(DataMappingTestSetup dataMappingTestSetup)
                 .add(new DataMappingTestSetup("smallint", "32123", "32767"))
                 .add(new DataMappingTestSetup("integer", "1274942432", "2147483647"))
                 .add(new DataMappingTestSetup("bigint", "312739231274942432", "9223372036854775807"))
-                .add(new DataMappingTestSetup("real", "REAL '567.123'", "REAL '999999.999'"))
-                .add(new DataMappingTestSetup("double", "DOUBLE '1234567890123.123'", "DOUBLE '9999999999999.999'"))
```

Comment:
I intentionally picked a value with 16 decimal digits, to verify enough of precision is retained.
I suggest we change `1234567890123.123` to `1234567890123.125`

Or, that we leave this as is, with the intent that connector doing some rounding, or lacks some precision, will exclude these test cases and provide some "easier" test cases instead.